data/ideas.json

```
@@ -220,6 +220,11 @@
         "description": "App for share knowledge by donationg books.",
         "level": "intermediate",
         "tags": "book-donation"
+    },
+    {
+        "title": "Barber Shop",
+        "description": "Reservations system made with Vue + Firebase ecosystem (cloud functions, auth, firestore, hosting).",
+        "level": "intermediate",
+        "tags": "vuejs, firebase, html, js, css"   
```

The first tag is used as a unique key to match the idea and project and remove technology because you could create it on another language
```suggestion
        "tags": "barber-shop"   
```